app/ts/constants.ts

```diff
@@ -1,3 +1,27 @@
-export const NETWORKS: { [chainId: string]: { simulationRelay: string, submissionRelay: string, blockExplorer: string, rpcUrl: string }} = {
-	'1': { simulationRelay: 'https://flashbots-cors-proxy.dark-florist.workers.dev/', submissionRelay: 'https://rpc.titanbuilder.xyz', blockExplorer: 'https://etherscan.io/', rpcUrl: 'https://rpc.dark.florist/flipcardtrustone' },
+type Network = { simulationRelay: string, submissionRelay: string, blockExplorer: string, rpcUrl: string, chainId: string, blockExplorerApi: string }
+
+export const MAINNET = {
```

Comment:
just to be explicit with type
```suggestion
export const MAINNET: Network = {
```

Reply:
the mapping below would already fail if this doesn't match?